### ISSUE TITLE
Run scheduler via WP-CLI

### ISSUE THREAD

**Issue by sc0ttkclark** [Prospress/action-scheduler#84]
Would be great to be able to run the scheduler via WP-CLI and get some better messaging as it processes each task, letting you go through all the tasks in the queue, and telling you how many tasks are left (using progress bar).

For the interim, I've been trying something like:
`wp eval "do_action( 'action_scheduler_run_queue' );"`

Though I'm not 100% sure if that works.

**Comment by thenbrent** [comment]
Nice idea.

> Though I'm not 100% sure if that works.

That should work, as long as `ActionScheduler_QueueRunner::init()` is executed and attached `ActionScheduler_QueueRunner::run()` to `'action_scheduler_run_queue'` (I'm not 100% sure of the order of operations of `wp eval`).

The main requirements to add this feature I think would be:

- [ ] add a command to run the queue
- [ ] output status messages when processing the queue, including:
    - [ ] status of each action: `started`/`complete`/`failed`, similar to the current log entries
    - [ ] status of queue: how many tasks are left

**Comment by sc0ttkclark** [comment]
Merry christmas:
https://gist.github.com/sc0ttkclark/6340930e13b3be76e0491e4148711ba3

Just fill it in :)

**Comment by sc0ttkclark** [comment]
To include the CLI command, just add:

```
if ( defined( 'WP_CLI' ) && WP_CLI ) {
    // Include the command file
}
```

**Comment by thenbrent** [comment]
@sc0ttkclark in case you haven't already seen it, there's work being done atm on this in https://github.com/Prospress/action-scheduler/pull/89

**Comment by thenbrent** [comment]
Implemented with #89
